packages/scss/src/components/textfield/mods.scss

```diff
@@ -241,12 +254,24 @@
 		line-height: var(--sizes-XS-lineHeight);
 		font-size: var(--sizes-XS-lineHeight);
 		bottom: var(--spacings-XXS);
+		right: var(--spacings-XXS);
 	}
 }
 
 @mixin searchClearableXS {
+	.textfield-input {
+		padding-right: 3rem;
+	}
+
 	.textfield-actionClear {
-		right: 1.25rem;
+		right: 1.75rem;
+		bottom: .375rem;
+		width: .75rem;
+		height: .75rem;
+
+		.lucca-icon {
+			font-size: .75rem;
+		}
```

Comment:
There are a lot of values set statically here, it doesn't feel highly maintainable.

Reply:
Yep, unfortunately it would be a XXS `line-height` and we don't have this token.
It's planned to rework sizing system to fit with new guidelines anyway.